libraries/helpers.rb

```diff
@@ -147,7 +147,12 @@ def php_pecl
       def php_version
         case node['platform_family']
         when 'rhel'
-          '7.2'
+          case node['platform_version'].to_i
+          when 9
+            '8.0'
+          else
+            '7.2'
+          end
```

Comment:
Id recommend doing if statement and comparing the version so that anything greater than 9 would get 8.0 to be more future proof for new OS releases 
```suggestion
          if node['platform_version'].to_i >= 9
            '8.0'
          else
            '7.2'
          end
```